coderd/provisionerjobs.go

```diff
@@ -334,98 +221,223 @@ func convertProvisionerJob(provisionerJob database.ProvisionerJob) codersdk.Prov
 	return job
 }
 
-func provisionerJobLogsChannel(jobID uuid.UUID) string {
-	return fmt.Sprintf("provisioner-log-logs:%s", jobID)
+func fetchAndWriteLogs(ctx context.Context, logger slog.Logger, db database.Store, jobID uuid.UUID, after int64, rw http.ResponseWriter) {
+	logs, err := db.GetProvisionerLogsAfterID(ctx, database.GetProvisionerLogsAfterIDParams{
+		JobID:        jobID,
+		CreatedAfter: after,
+	})
+	if err != nil && !errors.Is(err, sql.ErrNoRows) {
+		httpapi.Write(ctx, rw, http.StatusInternalServerError, codersdk.Response{
+			Message: "Internal error fetching provisioner logs.",
+			Detail:  err.Error(),
+		})
+		return
+	}
+	if logs == nil {
+		logs = []database.ProvisionerJobLog{}
+	}
+
+	logger.Debug(ctx, "Finished non-follow job logs")
```

Comment:
If you're planning on leaving this debug log line in it should have more details as fields, otherwise it should be removed

Reply:
caller adds job_id to the fields.  Not sure what else to add... `after`?